pom.xml

```diff
@@ -260,6 +260,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.directory.api</groupId>
+            <artifactId>api-util</artifactId>
```

Comment:
Have commented in the code but would be nice if we could avoid adding a dependency on the Apache APIs directly, especially if it is only for something small.